docs/usage/pvt.rst

```diff
@@ -67,21 +67,20 @@ Possibly, different viscosity scaling pr. PVTNUM is needed
 
 Density values are easier to scale up or down to whatever is needed.
 
-Re-exporting tables to Eclipse include files
+Re-exporting tables to include files
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
shorten line 71 (maybe there is a rstcheck in the CI that catches this?)

Reply:
rstcheck is there but I also ran that locally yesterday and it didn't catch the mismatch then